### PR TITLE
Import/05 triatlon

### PR DESCRIPTION
**Pull request by BlueEnni**
@TomasKostadinov can you take a final look at it and merge it - thanks :)

### CODE REVIEW

File: structure/04 Triathlon/01Trainer/2019_10_02TriathlonTrainer.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<html>
```
Comment (TomasKostadinov) [inline structure/04 Triathlon/01Trainer/2019_10_02TriathlonTrainer.html]:
```suggestion
<html>
<body>
```

Please create more valid html and include body tags 😉

File: structure/04 Triathlon/01Trainer/2019_10_02TriathlonTrainer.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+<h2>Triatlon</h2>
+<h6>Trainer</h6>
+<p>Triathlon-Trainer sind Anka Röhr und Sven Müller für das Schwimmen. Für das Radfahren gibt es keinen Trainer. Hier
+    trifft man sich meist am Wochenende nach Absprache über What’s App. Gelaufen wird im regulären Lauftreff mit den
+    entsprechenden Trainer. Für alles weitere siehe Lauftreff und Team.</p>
+
```
Comment (TomasKostadinov) [inline structure/04 Triathlon/01Trainer/2019_10_02TriathlonTrainer.html]:
```suggestion
</body>
```

**Review by TomasKostadinov — CHANGES_REQUESTED** [review]
Please add `body` tags to your files  - looks good otherwise!

**Comment by BlueEnni** [comment]
added html body tags @TomasKostadinov

**Comment by BlueEnni** [comment]
In my current view all tags are just fine - @TomasKostadinov can you confirm that?

**Review by TomasKostadinov — CHANGES_REQUESTED** [review]
Why do we need triathlon.html? 

Its an outdated version of elements.html
Please delete it.

**Comment by BlueEnni** [comment]
> Why do we need triathlon.html?
> 
> Its an outdated version of elements.html
> Please delete it.

Done.

**Comment by TomasKostadinov** [comment]
Finally!

**Review by TomasKostadinov — APPROVED** [review]

**Pull request merged**